### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by afestein**
Add more information about available options and defaults.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          bump-version: patch     # defaults to not bumping the package version
-          package-manager: yarn   # defaults to 'npm'
+          bump-version: patch # defaults to not bumping the package version
+          package-manager: yarn # defaults to 'npm'
+          git-user-email: my@email.com # defaults to '41898282+github-actions[bot]@users.noreply.github.com'
+          git-user-name: Test # defaults to 'github-actions[bot]'
+          pre-commit-script: ls
```
Comment (neverendingqs) [inline README.md:29]:
```suggestion
          pre-commit-script: npm run build # defaults to not running anything
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          bump-version: patch     # defaults to not bumping the package version
-          package-manager: yarn   # defaults to 'npm'
+          bump-version: patch # defaults to not bumping the package version
+          package-manager: yarn # defaults to 'npm'
+          git-user-email: my@email.com # defaults to '41898282+github-actions[bot]@users.noreply.github.com'
```
Comment (neverendingqs) [inline README.md:27]: Slight preference for `example.com`, as it is a reserved domain name as per https://tools.ietf.org/html/rfc2606#section-3 and can never actually resolve to someone's email servers.

```suggestion
          git-user-email: myemail@example.com # defaults to '41898282+github-actions[bot]@users.noreply.github.com'
```

File: README.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ jobs:
...
+          git-user-name: Test # defaults to 'github-actions[bot]'
+          pre-commit-script: ls
+          pull-request-labels: test # defaults to 'dependencies'
```
Comment (neverendingqs) [inline README.md:30]: If you're up for it, could you make sure all the comments start at the same spot? If not, I can fix that up after we merge this.

**Review by neverendingqs — CHANGES_REQUESTED** [review]
Thanks! If you have no objections, I would like to apply my suggested changes before merging.

**Comment by afestein** [comment]
Thanks for the review @neverendingqs, updated.

**Review by neverendingqs — APPROVED** [review]
Thanks!

**Pull request merged**